Documentation/network/external-workloads.rst

```diff
@@ -201,7 +186,7 @@ Next you can check the status of the Cilium agent in your external workload:
 
 .. code-block:: shell-session
 
-    cilium status
+    sudo cilium status
```

Comment:
wasn't sure how to document this. in my test setup i needed to do `sudo` to connect to /var/run/cilium/cilium.sock, but i guess it depends on how you log in to your external workload 💭

Reply:
I'm not sure we have any particular convention, this looks acceptable to me.